client/components/theme/index.jsx

```diff
@@ -123,18 +123,22 @@ const Theme = React.createClass( {
 	render() {
 		const {
 			name,
-			price,
 			screenshot
 		} = this.props.theme;
 		const {
 			active,
-			purchased
+			price,
+			translate
 		} = this.props;
 		const themeClass = classNames( 'theme', {
 			'is-active': active,
 			'is-actionable': !! ( this.props.screenshotClickUrl || this.props.onScreenshotClick )
 		} );
 
+		const priceClass = classNames( 'theme-badge__price', {
+			'theme-badge__price-plan': price === translate( 'Plan' )
```

Comment:
Should I be seeing a price of 'Plan'?

<img width="683" alt="screen shot 2017-06-30 at 11 59 52" src="https://user-images.githubusercontent.com/7767559/27733000-d631c394-5d8b-11e7-9bca-4bdc87bc6130.png">